lib/facebook/setup.js

```diff
@@ -69,13 +69,14 @@ module.exports = function fbSetup(api, bot, logError, optionalParser, optionalRe
             console.log(`\n\n${color.green}Facebook Messenger setup${color.reset}\n`);
             console.log(`\nFollowing info is required for the setup, for more info check the documentation.\n`);
             console.log(`\nYour webhook URL is: ${color.cyan}${lambdaDetails.apiUrl}/facebook${color.reset}\n`);
-            console.log(`Your verify token is: ${color.cyan}${token}${color.reset}\n`);
+            console.log(`   fda Your verify token is: ${color.cyan}${token}${color.reset}\n`);
```

Comment:
What is `fda`?

Reply:
Thought I deleted this in the second commit. Just a typo! That shouldn't be there

Reply:
Can you remove it?

Reply:
Sorry I missed this - done!